react/ProductSummaryListWithoutQuery.js

```diff
@@ -18,6 +18,10 @@ const List = ({ children, products }) => {
       products.map((product) => {
         const normalizedProduct = mapCatalogProductToProductSummary(product)
 
+        if (typeof ProductSummary === 'function') {
```

Comment:
Just out of curiosity, is it always needed to make this validation when dealing with Slots?

Reply:
@icazevedo I did it just to make sure is what I'm expecting. I think that if someone is using the component right now, and he/she doesn't have the types of the app he/she might think that it should pass an object (like `props.children`) instead of a component, that's why I'm doing this verification. In the end, I'm just trying to be extra safe